CHANGELOG.md

```diff
@@ -17,6 +17,14 @@
 * Refactor crypto code for cpp integration and add api to generate ephemeral asymmetric key pair. #1018
 * Update logger from Identity Core. (#1009)
 
+## [1.1.8] - 2020-08-20
+### Added
```

Comment:
Is the version expected to be here?

Reply:
good question. Let me find out. I just found out about this new changelog requirement myself.

Reply:
This should go to 1.1.9 release instead